includes/localization.php

```diff
@@ -56,3 +56,41 @@ function pmpro_translate_billing_period($period, $number = 1)
 			return __("Years", 'paid-memberships-pro' );
 	}
 }
+
+/**
+ * Handle translation updates from our own translation server.
+ * @since TBD
+ */
+function pmpro_check_for_translations() {
+	// Run it only on a PMPro page in the admin.
+	if ( ! current_user_can( 'update_plugins' ) ) {
+		return;
+	}
+
+	$is_pmpro_admin = ! empty( $_REQUEST['page'] ) && strpos( $_REQUEST['page'], 'pmpro' ) !== false;
```

Comment:
We do a similar check here: https://github.com/strangerstudios/paid-memberships-pro/blob/dev/includes/admin.php#L327-L331

Maybe we should have a function in includes/admin.php or adminpages/functions.php for this?

Reply:
I like the idea of having an admin function that checks is a pmpro setting page. We're starting to do this more.